code/modules/mob/living/carbon/human/species_types/roguetown/golem/golem.dm

```diff
@@ -108,3 +109,101 @@
 
 	))
 
+
+/datum/species/golem/on_species_gain(mob/living/carbon/C, datum/species/old_species)
+	. = ..()
+	C.construct = TRUE
+
+/datum/species/golem/on_species_loss(mob/living/carbon/C)
+	. = ..()
+	C.construct = FALSE
+
+
+//golem upgrade item so they can gain skills
+/obj/item/golem_skill_core
+	icon = 'icons/roguetown/items/misc.dmi'
+	name = "golem skill exhibitor"
+	desc = "A series of gears joined around a golden rod. When inserted into a Golem's head, it will allow them to grow their skills beyond their original design."
```

Comment:
```suggestion
	desc = "A series of gears joined around a copper rod. When inserted into a Golem's head, it will allow them to grow their skills beyond their original design."
```